uat/testing-features/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -9,25 +9,6 @@ Feature: Greengrass V2 LogManager
         And 5 temporary rotated log files for component aws.greengrass.Nucleus have been created
         And 5 temporary rotated log files for component UserComponentA have been created
 
-    @local
-    Scenario: LogManager-1-T0: install the local custom component
```

Comment:
Deleted this test because it is not actually testing an acceptance test. It was just running and testing the test component that will be used for testing the log manager